index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+
+    <title>Portfolio setup and mobile version skeleton</title>
+</head>
+<body>
+
+<header>
+
+  <nav>
+    <ul>
+     <li id="logo"><a href="#"><img src="./images/logo.png" alt="Website logo"></a></li>
+     <li id="menu"><a href="#"><img src="./images/menu.png" alt="Website menu"></a></li> 
+    </ul>
+  </nav>
+</header>
+
+<section class="headline">
+  <div class="header-wrapper">
+
+  <h1 id="greeting">Hey there! I'm Mariane.</h1>
+  <h2 class="introduction">I'm a software developer</h2>
+  <p class="text">I can help you build a product, feature or website. 
+    Look through some of my work and experience.If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+
+    <ul class="social-handle">
+        <li><a href="#"><img src="./images/twitter.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/Linkedin-icon.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/medium.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/github.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/angellist.png" alt=""></a></li>
+    </ul>
+    <div class="button">
+        <a href="#"><img src="./images/Disabled.png" alt="Button image"></a>
+   </div>
+   
+  </div>
+</section>
+
+<section class="works">
```

Comment:
- [ ] Kindly center all cards  so it looks like the Figma design, you can use `margin: 0 auto`👍🏿 
![Screenshot_89](https://user-images.githubusercontent.com/69638013/165524251-a2517cae-58d6-4b31-98d4-f5fb39535766.png)